base.py

```diff
@@ -1323,6 +1334,11 @@ def get_local_backup(relative_path, rev='latest'):
     GNP_SCRIPT =  format_slash.__func__('%s/misc/gnp.py' % PROJECT_TOOLKIT_DIR)
     MESA_SCRIPT = format_slash.__func__('%s/misc/mesa.py' % PROJECT_TOOLKIT_DIR)
 
+    # We cannot ensure that all users on a host have put their ssh public keys to backup server,
+    # so use a shared key instead of personal key to allow the service to always access the server,
+    # no matter which user logs in.
+    SSH_KEY = format_slash.__func__('%s/ssh_key/id_rsa' % PROJECT_DIR)
```

Comment:
I think we can put it in TOOL_DIR, as it's not a project.

Reply:
The projects using util will have different TOOL_DIRs, we need to put the keys to each project, but I just want to keep them in one place on a host.
Or we upload the keys to util/tool, which would violate security policy, this is a public repo and we'd exposed some server names in it.

If PROJECT_DIR is not a good place, how about PROJECT_TOOLKIT_DIR where it's really used.

Reply:
One more point, other projects may also require the key, such as ort test. The path for the keys should be as universal as possible.

Reply:
You're right. I was thinking of some folder in "d:\workspace\tools". Folder project supposes to host all the git repos. I did want to bring in a folder like this, but I changed the mind later on to check in the small tools to repo util to save maintenance cost.

Reply:
Can we just use repo work, and put private key at config/common/? In this way, we don't need any extra setup on various machines. I really hope we can automatically maintain all the machines, including the introduction of new machines.

Reply:
Ok, I will upload the key to work/config/common, then automatically copy to each user's .ssh directory, we will use the key copied there.
Also defined a `SSH_DIR` to check whether the path exists when copying the file.